kernel/src/hil/screen.rs

```diff
@@ -90,6 +116,14 @@ impl ScreenPixelFormat {
     }
 }
 
+/// Returned for combined operations
+pub enum UpdateType {
+    /// All settings have been applied
+    Complete,
+    /// Not all settings have been applied
+    Partial,
+}
+
```

Comment:
I think you can delete this as it is not used anymore.

Reply:
Thanks for noticing that. It's gone.